modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -0,0 +1,85 @@
+[[exportword-service-url]]
+== `exportword_service_url`
+
+By default, the Word export feature uses the {productname} Cloud Services HTML to DOCX converter service to generate the Word files. You can use this option to provide the URL to an on-premises converter.
+
+*Type:* `+String+`
+
+=== Example: using `exportword_service_url`
+
+// Add a working and tested configuration.
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportword',
+  toolbar: 'exportword',
+  exportword_service_url: '<service_URL>'
+});
+----
+
+[[exportword-converter-options]]
+== `exportword_converter_options`
+
+The `exportword_converter_options` option, is used to specify and configure various options related to the process of converting and exporting content from the editor to `.docx` Word file.
+
+*Type:* `+Object+`
+
+*Default value:* `{}`
+
+=== Example: using `exportword_converter_options`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportword',
+  toolbar: 'exportword',
+  exportword_service_url: '<service_URL>',
+  exportword_converter_options: {
+    header: [
+      {
+        html: '<h1>First page header.</h1>',
+        css: 'h1 { font-size: 30px; }',
+      }
+    ],
```

Comment:
There are other options as well, the full list of options and examples is listed [here](https://exportdocx.converter.tiny.cloud/docs#options)

Reply:
added as above suggestion, im not 100% happy with these being links, but it might have to do for initial launch.